src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -50,7 +50,7 @@ class File extends Constraint
     public $uploadFormSizeErrorMessage = 'The file is too large.';
     public $uploadPartialErrorMessage = 'The file was only partially uploaded.';
     public $uploadNoFileErrorMessage = 'No file was uploaded.';
-    public $uploadNoTmpDirErrorMessage = 'No temporary folder was configured in php.ini.';
+    public $uploadNoTmpDirErrorMessage = 'No temporary folder was configured in php.ini, or the configured folder does not exist.';
```

Comment:
Ah, I didn't know that was a possibility. 

I've updated the PR; only the language files for EN, NL and PL now have the change.